isTemperatureInRange.c

```diff
@@ -0,0 +1,12 @@
+#include "header.h"
+
+int isTemperatureInRange(float temperature)
+{
+    Std_ReturnType RetVal = E_OK;
+    if(temperature < 0 || temperature > 45) 
```

Comment:
Avoid repeated < || > checks by having them in a function e.g., `isWithinRange`. Pass the range to it as well.
This way only that function needs to be tested and re-used everywhere.